### PR TITLE
fix URLs

### PR DESCRIPTION
**Pull request by maximsmol**

### CODE REVIEW

File: wf/__init__.py [hunk 1]
```diff
@@ -325,9 +362,14 @@ def _flag(name: str) -> List[str]:
         raise TrimgaloreError(stdout)
 
     def remote(middle: str) -> str:
```
Comment (AidanAbd) [inline wf/__init__.py:364]: This parameter name upsets me.

File: wf/__init__.py [hunk 2]
```diff
@@ -84,6 +93,39 @@ def ___repr__(self):
...
+
+    Paths are always joined, with extra `/`s added if missing. Does not allow
+    overriding basenames as opposed to normal `urljoin`. Whether the final
+    path ends in a `/` is still significant and will be preserved in the output
+
+    >>> urljoin("latch:///directory/", "another_directory")
+    latch:///directory/another_directory
+    >>> # No slash means "another_directory" is treated as a filename
+    >>> urljoin(urljoin("latch:///directory/", "another_directory"), "file")
+    latch:///directory/file
```
Comment (AidanAbd) [inline wf/__init__.py:107]: This makes no sense

**Review by AidanAbd — CHANGES_REQUESTED** [review]
Some sussery

File: wf/__init__.py (hunk 1 above)
Comment (maximsmol) [reply wf/__init__.py:364]: not my code 🙃

**Pull request merged**